compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -102,10 +102,7 @@ enum class VectorPreProcStrategy {
   // be masked-out.
   Masking,
   // Do not apply any vectorization pre-processing transformation.
-  None,
-  // A hint for the compiler to use its heuristics to determine an
-  // actual pre-processing strategy.
-  Heuristics
+  None
 };
 
 static llvm::cl::opt<VectorPreProcStrategy> clPProcStrategy(
```

Comment:
The change in new revision is new to me. I was not aware of this trick. Can you add a comment here? People might ask why default is None, but it is not using None. Otherwise, they have to look into the implementation to understand what's happening.

Reply:
I also wasn't aware of this, but if you grep for `getNumOccurrences()` in LLVM then you'll see that it's quite a common idiom 🤷🏻  :) 

There isn't that much in terms of documentation. This is the best that I could find: https://github.com/llvm/llvm-project/blob/e3172e841899779caf461cee97826940dca806e7/llvm/include/llvm/Support/CommandLine.h#L274

I will remove the "init" value - we are not using it anyway. Not sure what to add in terms of comments. Will send a small update shortly - let me know what you think.

Reply:
LGTM, thank you!